editor/editor_help.h

```diff
@@ -75,6 +75,7 @@ class EditorHelpSearch : public ConfirmationDialog {
 class EditorHelpIndex : public ConfirmationDialog {
 	GDCLASS(EditorHelpIndex, ConfirmationDialog);
 
+	EditorNode *editor;
```

Comment:
I don't think we need a pointer to the editor singleton here. Most Godot code uses `EditorNode::get_singleton()` directly, and in EditorHelp it's only used once, so really not worth saving to a member var IMO.